### PR TITLE
Users/suaggar/webdeploytlsguidancemsg

### PR DESCRIPTION
**Pull request by SumiranAgg**
Guidance message for TLS error in web deploy

### CODE REVIEW

File: Tasks/IISWebAppDeploymentOnMachineGroup/task.json [hunk 1]
```diff
@@ -179,6 +179,7 @@
...
         "FailedToApplyTransformationReason2": "2. Ensure that the config file and transformation files are present in the same folder inside the package.",
         "AutoParameterizationMessage": "ConnectionString attributes in Web.config is parameterized by default. Note that the transformation has no effect on connectionString attributes as the value is overridden during deployment by 'Parameters.xml or 'SetParameters.xml' files. You can disable the auto-parameterization by setting /p:AutoParameterizationWebConfigConnectionStrings=False during MSBuild package generation.",
-        "PackageDeploymentFailed": "Failed to deploy web package to IIS website."
+        "PackageDeploymentFailed": "Failed to deploy web package to IIS website.",
+        "Updatemachinetoenablesecuretlsprotocol" : "Stronger security protocol might be required. Please make sure the machine is using TLS 1.1 or higher. Check the link for more information https://support.microsoft.com/en-us/help/3206898/enabling-iis-manager-and-web-deploy-after-disabling-ssl3-and-tls-1-0"
```
Comment (vincentdass) [inline Tasks/IISWebAppDeploymentOnMachineGroup/task.json]: remove 'please'

File: Tasks/Common/webdeployment-common/msdeployutility.ts [hunk 2]
```diff
@@ -195,6 +195,9 @@ export function redirectMSDeployErrorToConsole() {
             else if(errorFileContent.indexOf("FILE_IN_USE") !== -1) {
                 tl.warning(tl.loc("Trytodeploywebappagainwithrenamefileoptionselected"));
             }
+            else if(errorFileContent.indexOf("Unable to read data from the transport connection: An existing connection was forcibly closed by the remote host.") != -1){
```
Comment (vincentdass) [inline Tasks/Common/webdeployment-common/msdeployutility.ts]: Can we reduce the length of the hard-coded message?

Comment (SumiranAgg) [reply Tasks/Common/webdeployment-common/msdeployutility.ts]: parsing just keyword now

File: Tasks/IISWebAppDeploymentOnMachineGroup/task.json (hunk 1 above)
Comment (SumiranAgg) [reply Tasks/IISWebAppDeploymentOnMachineGroup/task.json]: Changed error message

[the remark this answers is not in the document]
Comment (SumiranAgg) [reply Tasks/IISWebAppDeploymentOnMachineGroup/task.json]: replaced by forward link

**Pull request merged**